pkgs/development/python-modules/m2r/default.nix

```diff
@@ -38,7 +38,7 @@ buildPythonPackage rec {
     homepage = "https://github.com/miyakogi/m2r";
     description = "Markdown to reStructuredText converter";
     license = licenses.mit;
-    maintainers = with maintainers; [ SuperSandro2000 ];
+    maintainers = with maintainers; [ AndersonTorres SuperSandro2000 ];
```

Comment:
What am I doing here??

Reply:
You were a maintainer of the removed `pkgs/tools/text/m2r`.

Reply:
@AndersonTorres Do you want me to remove your name?